### ISSUE TITLE
Figma access request

### ISSUE THREAD

**Issue by mengA6** [department-of-veterans-affairs/va.gov-team#71702]
### Name

Mike Eng

### Email

Please see [my slack profile](https://dsva.slack.com/team/U066PHHJGNQ) (trying to avoid exposing myself to spam on this public issue)

### Role

UX Designer / Researcher

### Team

Disability Benefits Experience Team 1

### Product Manager (PM) name

Shannon Ford

### Additional Notes

I currently have edit access in Sketch but am not seeing a Figma invite in either my contractor email or VA email. I'm fairly new here, so might not be on the appropriate lists yet.

**Comment by mengA6** [comment]
Thank you! I'm in but not sure I have the correct permissions. I can view the "Disability Benefits" project but don't see buttons to create or import a design file.

![Screenshot 2023-12-11 at 3 54 59 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/151068099/eeeda08b-1852-4440-bdca-70a0124978c0)

**Comment by naomimarcussen** [comment]
You should have editor access now. Sorry your name got lost in the transition. This might take a few minutes, so if you don't see access wait a few. :)

**Comment by mengA6** [comment]
No problem.

Checking back here, I still don't think I have edit access. I see the same thing (no buttons) with the Disability Benefits project open, even after I logged out and logged back in on
<img width="1512" alt="Figma with Disability Benefits project open - no create file or import buttons visible" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/151068099/d67f94a4-c490-48b9-89ca-dfc8b06b71a6">
 the Mac Desktop app.

And when I tried to drag a Sketch file onto the Figma project, Figma didn't respond ([screen recording](https://jmp.sh/EEaBSm9V))
